common/src/main/java/io/netty/util/AsciiString.java

```diff
@@ -714,6 +714,34 @@ public int indexOf(CharSequence subString, int start) {
     }
 
     /**
+     * Searches in this string for the index of the specified char {@code ch}.
+     * The search for the char starts at the specified offset {@code start} and moves towards the end of this string.
+     *
+     * @param ch the char to find.
+     * @param start the starting offset.
+     * @return the index of the first occurrence of the specified char {@code ch} in this string,
+     * -1 if found no occurrence.
+     */
+    public int indexOf(char ch, int start) {
+        if (start < 0) {
```

Comment:
arg ... I guess `String.indexOf(..)` does this and you are just trying to be consistent?